code/modules/projectiles/guns/projectile/pistol.dm

```diff
@@ -16,7 +16,7 @@
 	return
 
 /obj/item/weapon/gun/projectile/automatic/pistol/m1911
-	name = "\improper M1911 pistol"
+	name = "M1911"

```

Comment:
This should keep pistol at the end. That was done EXACTLY for the reason so you could easily see what type of gun it is or what the heck was shot at you.